Mathlib/Data/List/Rotate.lean

```diff
@@ -247,8 +247,8 @@ theorem get?_rotate {l : List α} {n m : ℕ} (hml : m < l.length) :
         rw [tsub_lt_iff_left hm']
         exact Nat.add_lt_add hlt hml
       conv_rhs => rw [add_comm m, ← mod_add_mod, mod_eq_sub_mod hm', mod_eq_of_lt this]
-      rw [← add_right_inj l.length, ← add_tsub_assoc_of_le, add_tsub_tsub_cancel,
-        add_tsub_cancel_of_le, add_comm]
+      rw [← add_right_inj l.length, ← add_tsub_assoc_of_le (α := ℕ), add_tsub_tsub_cancel (α := ℕ),
```

Comment:
Why do these now need an implicit argument?

Reply:
I am not entirely sure; apparently you can do this proof without some imports and it still works when you pass the types explicitly but inferring them seems not to work, or you can add more imports and it will work as is. I felt like it would be better to err on the side of fewer imports since the fallout isn't too much.